packages/manager-react-components/src/components/content/dashboard-tile/help-icon-with-tooltip.tsx

```diff
@@ -0,0 +1,31 @@
+import { useId } from 'react';
+import { ODS_TEXT_PRESET } from '@ovhcloud/ods-components';
+import { OdsText, OdsIcon, OdsTooltip } from '@ovhcloud/ods-components/react';
+
+type HelpIconWithTooltipProps = {
+  tooltip: string;
```

Comment:
```suggestion
  tooltipLabel: string;
```
it'll be clearer that way